ppr-ui/src/components/mhrRegistration/ReviewConfirm/SubmittingPartyReview.vue

```diff
@@ -91,7 +91,7 @@
 
           <v-row no-gutters class="px-6 py-7">
             <v-col cols="3">
-              <p class="side-header">Attention or<br>Reference Number</p>
+              <p class="side-header">Attention <span v-if="!isRoleStaffReg"> or<br>Reference Number</span></p>
```

Comment:
Can we use a `title` from a newly `attnRefConfig` here?

Reply:
Yes! 
Definitely, good eye. Will fix